docs/conf.py

```diff
@@ -1,69 +1,81 @@
-# Configuration file for the Sphinx documentation builder.
-#
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""Configuration file for the Sphinx documentation builder.
 
-# -- Path setup --------------------------------------------------------------
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
+This file only contains a selection of the most common options. For a full
+list see the documentation:
+https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""
 
 import os
-import sys
+import shutil
+import subprocess
 
-import tf_pwa
 from tf_pwa.amp import PARTICLE_MODEL, get_config
-from tf_pwa.experimental import extra_amp
```

Comment:
It import some Resonances model in experimental.

Reply:
But where is the imported `extra_amp` used in `conf.py`?

Reply:
To generate documents of those model

Reply:
Importing extra_amp will touch PARTICLE_MODEL to add a model. So, get_config(PARTICLE_MODEL) can get those Model

Reply:
Ah ok I see. Because importing those modules registers new dynamic models. Alright, I added it back (c030b6a), although this seems a bit like code smell to me.

Reply:
Have a look at the PR preview on RTD (nice that it works!), for me the experimental resonances are visible when I build the docs locally.

Reply:
Works!
https://pwa--7.org.readthedocs.build/projects/tf-pwa/en/7/particle_model.html
And here you can see the purpose of this PR: you can just click the links to go to the definition in the API and even check out the source code there